addons/mail/static/src/messaging/component/message/message.scss

```diff
@@ -97,6 +97,10 @@
     font-size: 0.8em;
 }
 
+.o_Message_seenIndicator.o-message-squashed {
+    position: absolute;
+}
```

Comment:
That's kinda weird that the component itself should be absolutely positioned and container must itself be aware of that and be relatively positioned. Haven't yet see what's the underlying problem that motivated this style rule.

Reply:
That's because the date part is not removed (the part that is shown on hover on squashed message) when not hovered, so without that part, the indicator is placed after it and then totally on the right in the message body part

Reply:
Ok, then data should disappear from DOM on hover, so hover should be managed logically in template.